### PR TITLE
Doc: update development instructions

### PR DESCRIPTION
**Pull request by bzz**
Minor drive-by update of the README to avoid confusing first-time users/developers giving it a try.
It also documents minimum requirements.

### CODE REVIEW

**Comment by bzz** [comment]
Will rebase this on #82 as soon as it gets merged

File: README.md [hunk 1]
```diff
@@ -12,21 +12,29 @@ docker run -p 8080:80 bblfsh/dashboard -bblfsh-addr <bblfsh-server-addr>
...
 ```sh
-docker run --privileged -d -p 9432:9432 --name bblfsh bblfsh/server
+docker run --privileged -d -p 9432:9432 --name bblfsh bblfsh/bblfshd
 docker run -p 8080:80 --link bblfsh bblfsh/dashboard -bblfsh-addr bblfsh:9432
 ```
 
-Please read the [getting started](https://doc.bblf.sh/user/getting-started.html) guide, to learn more about how to use and deploy a bblfsh server.
+Please read the [getting started](https://doc.bblf.sh/user/getting-started.html) guide, to learn more about how to use and deploy a bblfsh server, install drivers, etc.
 
 If don't want to run **dashboard** using our *Docker* image you can download a binary from [releases](https://github.com/bblfsh/dashboard/releases) page.
```
[the remark this answers is not in the document]
Comment (bzz) [reply README.md]: Sorry, I'm a bit confused - do you mean that a notice about bblfshd needs to be added to the last sentence, about binary releases?

**Comment by dpordomingo** [comment]
@bzz you added some useful info with this PR. Could you rebase over `src-d/master` and merge? :relaxed:

**Comment by bzz** [comment]
@dpordomingo sorry for delay. Done.

**Comment by bzz** [comment]
@dpordomingo friendly ping

**Comment by dpordomingo** [comment]
Merged, thanks for pinging

**Pull request merged**